### PR TITLE
Implement initial setup for group voting and seeing results

### PR DESCRIPTION
**Pull request by haiyingweng**
### Overview 
- setup VC to vote for show
- setup VC to view voting results
- added swipe gesture btw two VCs

### Screenshots/GIFs

vote
show info
result

### CODE REVIEW

File: Flick/Group/GroupResultViewController.swift [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import UIKit
+
+class GroupResultViewController: UIViewController {
+
+    // MARK: - Private View Vars
+    private let resultsTableView = UITableView()
+    private let votingStatusImageView = UIImageView()
+    private let votingStatusLabel = UILabel()
+
+    // MARK: - Private Data Vars
```
Comment (lucyxubroad) [inline Flick/Group/GroupResultViewController.swift:18]: can we start to follow new style of having the _reuseIdenfier_ as a static variable in the class file for the cell?

File: Flick/Group/GroupResultViewController.swift [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    private let votingResultCellReuseIdentifier = "votingResultCellReuseIdentifier"
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .offWhite
+
+        votingStatusImageView.image = UIImage(named: "stillVotingIcon")
```
Comment (lucyxubroad) [inline Flick/Group/GroupResultViewController.swift:25]: idk if this is better but I believe you can just do like  `private let votingStatusImageView = UIImageView(image: UIImage(named: "stillVotingIcon"))` I like this cause its more succinct but also like a lot of things all in one line

File: Flick/Group/GroupTabPageViewController.swift [hunk 3]
```diff
@@ -0,0 +1,69 @@
+//
```
Comment (lucyxubroad) [inline Flick/Group/GroupTabPageViewController.swift:1]: let's make a note to try and make a reusable view for our tab page view controllers! we definitely dont need this right now hahaha

File: Flick/Group/GroupViewController.swift [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+    // MARK: - Private View Vars
+    private var tabCollectionView: UICollectionView!
+    private var tabContainerView = UIView()
+    private let tabPageViewController = GroupTabPageViewController()
+
+    // MARK: - Private Data Vars
+    private var activeTabIndex = 0
+    private let tabBarHeight: CGFloat = 40
+    private let tabCellReuseIdentifier = "tabCellReuseIdentifier"
```
Comment (lucyxubroad) [inline Flick/Group/GroupViewController.swift:21]: same note about reuseIdentifier

File: Flick/Group/GroupVoteViewController.swift [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+    private let moreInfoView = UIStackView()
+    private let numIdeasLabel = UILabel()
+    private let posterImageView = UIImageView()
+    private let voteMaybeButton = UIButton()
+    private let voteNoButton = UIButton()
+    private let voteYesButton = UIButton()
+
+    // MARK: - Private Data Vars
+    private var ideas: [Media] = []
+    private let mediaSummaryReuseIdentifier = "MediaSummaryReuseIdentifier"
```
Comment (lucyxubroad) [inline Flick/Group/GroupVoteViewController.swift:25]: same note here

File: Flick/Group/GroupVoteViewController.swift [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+            make.size.equalTo(voteButtonSize)
+            make.bottom.equalTo(addIdeasButton.snp.top).offset(-30)
+        }
+    }
+
+    @objc private func longPressedPoster(sender: UILongPressGestureRecognizer) {
+        if sender.state == .began {
+            mediaInformationTableView.isHidden = false
+        } else if sender.state == .ended {
+            mediaInformationTableView.isHidden = true
```
Comment (lucyxubroad) [inline Flick/Group/GroupVoteViewController.swift:167]: should we add haptic feedback here?? :o

File: Flick/Group/VotingResultTableViewCell.swift [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+
+        voteMaybeImageView.image = UIImage(named: "voteMaybeEmoticon")
+        contentView.addSubview(voteMaybeImageView)
+
+        numVoteMaybeLabel.text = "5"
+        numVoteMaybeLabel.font = .systemFont(ofSize: 16)
+        numVoteMaybeLabel.textColor = .mediumGray
+        contentView.addSubview(numVoteMaybeLabel)
+
+        voteNoImageView.image = UIImage(named: "voteNoEmoticon")
```
Comment (lucyxubroad) [inline Flick/Group/VotingResultTableViewCell.swift:53]: I have the same note about all image views actually but you dont have to if you think it's wonky! just I know we do a lot of things based on appdev standards but that might not even be the best way to do things

**Review by lucyxubroad — APPROVED** [review]
this looks FANTASTIC and I'm so excited to have the groups feature omg

File: Flick/Group/GroupVoteViewController.swift (hunk 6 above)
Comment (haiyingweng) [reply Flick/Group/GroupVoteViewController.swift:167]: woah yea that'd be very cool

**Pull request merged**